docs/encryption.md

```diff
@@ -80,6 +80,10 @@ func EncryptImage(ctx context.Context, cs content.Store, desc ocispec.Descriptor
 // DecryptImage decrypts an image; it accepts either an OCI descriptor representing a manifest list or a single manifest
 func DecryptImage(ctx context.Context, cs content.Store, desc ocispec.Descriptor, cc *encconfig.CryptoConfig, lf *LayerFilter) (ocispec.Descriptor, bool, error)
 
+// DecryptLayer decrypts the layer using the CryptoConfig and creates a new OCI Descriptor.
+// The caller is expected to store the returned plain data and OCI Descriptor
+func DecryptLayer(cc *encconfig.CryptoConfig, dataReader content.ReaderAt, desc ocispec.Descriptor, unwrapOnly bool) (ocispec.Descriptor, io.Reader, error)
```

Comment:
From previous PR, i think there was a comment to make this also return the `digest.Digest` as well.

Reply:
Ok, added it.